Models/AuthenticationModels.cs

```diff
@@ -1,73 +1,78 @@
 ﻿using CommunifyLibrary.Models;
 
-namespace Communify_Backend.Models
+namespace Communify_Backend.Models;
+
```

Comment:
Generic comment about the entire file. Each file has only ONE class. I dont see any maxlength associated with the comments. Please give it a maxlength using [Maxlength(int)] or your modelbuilder.